### DISCUSSION TITLE
Newbie, INSTALL DEPENDENCIES?? won't run ;(

### DISCUSSION THREAD

**Discussion by renaoi** [community/community#108343] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have never used github before, what does the mean?

Run npm ci --ignore-scripts
  npm ci --ignore-scripts
  shell: /usr/bin/bash -e {0}
  env:
    BUILD_PATH: .
    NOTION_API_SECRET: ***
npm WARN deprecated abab@[2](https://github.com/renaoi/rennko.github.io/actions/runs/7908409265/job/21587519585#step:5:2).0.6: Use your platform's native atob() and btoa() methods instead
npm WARN deprecated domexception@4.0.0: Use your platform's native DOMException instead
npm WARN tarball tarball data for astro-tweet@https://gitpkg.now.sh/nerdymomocat-templates/astro-v4-tweet/packages/astro-tweet?main (sha512-uz[3](https://github.com/renaoi/rennko.github.io/actions/runs/7908409265/job/21587519585#step:5:3)7mrymQxNW0gAGEz8NNwXCAcdIwji+NZ+GZMeR5MwR2[4](https://github.com/renaoi/rennko.github.io/actions/runs/7908409265/job/21587519585#step:5:4)x01VoykOFcs2w83fD7gcwYoqo9VYNYkXy7YMW7[5](https://github.com/renaoi/rennko.github.io/actions/runs/7908409265/job/21587519585#step:5:5)A==) seems to be corrupted. Trying again.
npm ERR! code EINTEGRITY
npm ERR! sha512-uz37mrymQxNW0gAGEz8NNwXCAcdIwji+NZ+GZMeR5MwR24x01VoykOFcs2w83fD7gcwYoqo9VYNYkXy7YMW75A== integrity checksum failed when using sha512: wanted sha512-uz37mrymQxNW0gAGEz8NNwXCAcdIwji+NZ+GZMeR5MwR24x01VoykOFcs2w83fD7gcwYoqo9VYNYkXy7YMW75A== but got sha512-MDwr[6](https://github.com/renaoi/rennko.github.io/actions/runs/7908409265/job/21587519585#step:5:6)NBIrXq1JdIqeo[7](https://github.com/renaoi/rennko.github.io/actions/runs/7908409265/job/21587519585#step:5:8)Mt1XC4fF5ti7Swm7fJtlxkYcNC9aozXNIpc61KKqgbUdV0Q9Ofxi4fGqetXdLFBfwow==. (7546[8](https://github.com/renaoi/rennko.github.io/actions/runs/7908409265/job/21587519585#step:5:9) bytes)

npm ERR! A complete log of this run can be found in: /home/runner/.npm/_logs/2024-02-[14](https://github.com/renaoi/rennko.github.io/actions/runs/7908409265/job/21587519585#step:5:15)T22_51_14_030Z-debug-0.log
Error: Process completed with exit code 1.

**Comment by Ariffazeel99** [comment]
Hi,
you can solve this by following :
Clear the npm Cache and then retry the Installation
npm cache clean --force
npm ci --ignore-scripts
Inspect Logs: /home/runner/.npm/_logs/2024-02-14T22_51_14_030Z-debug-0.log
